frog/package.json

```diff
@@ -13,6 +13,12 @@
     "meteor-node-stubs": "~0.2.0",
     "react": "^15.3.2",
     "react-addons-pure-render-mixin": "^15.3.2",
+    "react-dom": "^15.3.2",
+    "react-player": "^0.12.0"
+  },
+  "scripts": {
+    "test": "meteor test --once --driver-package dispatch:mocha-phantomjs",
+    "start": "meteor run",
```

Comment:
This file is not correct, there must have been a problem when merging develop
- The field `scripts` appears twice, keep only the first one
- The field `react-dom` appears twice, must be only once inside the `dependencies`